ci/get-ocp-repo.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+echo test!
+ocpver=$(rpm-ostree compose tree --print-only manifest.yaml | jq -r '.["mutate-os-release"]')
+ocpver_mut=$(rpm-ostree compose tree --print-only manifest.yaml | jq -r '.["mutate-os-release"]' | sed 's|\.|-|')
+rhelver=$(rpm-ostree compose tree --print-only manifest.yaml | jq -r '.["automatic-version-prefix"]' | cut -f2 -d.)
+
+curl -L "http://base-${ocpver_mut}-rhel${rhelver}.ocp.svc.cluster.local" -o "ocp.repo"
```

Comment:
This is totally OK for now but maybe a quick:
`# TODO: share this with ci/prow-entrypoint.sh` ?

Reply:
TODO added. I can update the scripts when doing the COSA work as will probably need to change the Dockerfile then too.